evap/staff/importers.py

```diff
@@ -303,8 +306,26 @@ def write_enrollments_to_db(self, semester, vote_start_date, vote_end_date):
                 student = UserProfile.objects.get(email=student_data.email)
                 course.participants.add(student)
 
-        self.success_messages.append(_("Successfully created {} course(s), {} student(s) and {} contributor(s).").format(
-            len(self.courses), students_created, responsibles_created))
+        msg = _("Successfully created {} course(s), {} student(s) and {} contributor(s):").format(
+            len(self.courses), len(students_created), len(responsibles_created))
+        for user in students_created + responsibles_created:
+            msg += "<br>"
+            msg += "{} {} ({})".format(user.first_name, user.last_name, user.username)
+        self.success_messages.append(mark_safe(msg))
+
+    def create_test_success_messages(self):
+        filtered_users = []
+
+        for user_data in self.users.values():
+            if not user_data.user_already_exists():
+                filtered_users.append(user_data)
```

Comment:
filtered_users = [user_data for user_data in self.users.values() if not user_data.user_already_exists()]